addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -488,7 +488,7 @@ export class MassMailingHtmlField extends HtmlField {
                 class: 'container o_mail_wrapper o_mail_regular oe_unremovable',
             });
             $newWrapperContent = $('<div/>', {
-                class: 'col o_mail_no_options o_mail_wrapper_td bg-white oe_structure o_editable'
+                class: 'col o_mail_no_options o_mail_wrapper_td bg-white oe_structure o_editable oe_unremovable',
```

Comment:
@detrouxdev I don't think that enough or even the good solution: on pre-existing mailing on runbot I still have the problem. I think this "newWrapperContent" is for new mailing content but maybe you would still have the issue for mails coming from earlier version. The fact that a trash icon appears is already governed by some rules (the parent must be editable and maybe other stuff). I think you should check that. There is also the possibility to create a snippet option with the property `forceNoDeleteButton`... in the end we really should improve the framework behind all of this.